packages/mathtype-ckeditor5/README.md

```diff
@@ -1,12 +1,10 @@
-MathType for CKEditor 5 [![Tweet](https://img.shields.io/twitter/url/http/shields.io.svg?style=social)](https://twitter.com/wirismath)
+MathType for CKEditor 5
 ===
 
 Type and handwrite mathematical notation with MathType.
 
 Easily include quality math equations in your documents and digital content.
 
-![MathType for CKEditor 5 screenshot](http://www.wiris.com/portal/themes/wiris_com/img/others/ckeditor_plugin_npm.png)
-

```

Comment:
Even though removing the image is a solution to the broken link error, I believe it would be better to recover the original image.

Please do that.